chore: changelog 1.4.1 (#485)
plentymarkets/plentyshop-pwa: docs/changelog/changelog_en.md

# Changelog plentyshopPWA

## v1.4.1 (2024-06-04) <a href="https://github.com/plentymarkets/plentyshop-pwa/compare/v1.4.0...v1.4.1" target="_blank" rel="noopener">

### New

> - Add custom SVG icon support

Reviewer: ```suggestion
- Custom SVG icon support
```

> - Lazy-load images

Reviewer: ```suggestion
- Images outside the visible area are lazy loaded
```

> - A guest login page before going to checkout

Reviewer: ```suggestion
- Login dialog before checkout
```

> - Selection for the saved addresses at checkout

Reviewer: ```suggestion
- Selection of saved addresses in the checkout
```

- Width and height attributes to images on category and product view

> - Product reviews on single item pages

Reviewer: ```suggestion
- Product reviews on item pages
```

> - Added quick checkout functionality.

Reviewer: ```suggestion
-  “Item added to shopping cart” dialog
```

> - Added quantity to quick checkout product name

Reviewer: This is not relevant because its part of the  “Item added to shopping cart” dialog feature

> - Cookie bar translations.

Reviewer: See comment for german translation

### Fixed

> - Fixed coupon display on thankyou page

Reviewer: ```suggestion
- Voucher display on the order confirmation
```

- Fixed issues related to product reviews

> - Fixed thank-you page missing details

Reviewer: ```suggestion
- Added missing details on the order confirmation
```

- Fixed errors when interacting with wishlist

> - Fixed defineExpose compiler error

Reviewer: What is this

> - Fixed random Nuxt instance unavailable

Reviewer: What is this

- We do now ensure that there is no `/` at the end of the API_URL.
- Fixed facet filtering on category page
- Fixed random login error

> - Creating new item in basket, instead of updating quantity

Reviewer: ```suggestion
- When adding an item that is already in the shopping cart, the quantity is increased instead of adding the same item to the shopping cart again
```

> - Bundle item accessability via hyperlink, if not fully set

Reviewer 2: - Components of article packages without an article URL no longer lead to a 404 page. 
- Components of item packages without a maintained name now display the placeholder "Product Information Missing" in place of the name

Author: done

> - Fixed the filtering in SSR on the category page

Reviewer: ```suggestion
- Server-side rendering error due to filters on the category view
```

- Fixed hydration and intlify warnings
- The PayPal express button on the product page handles the click event correctly.
- Fixed aspect ratio on homepage hero img
- Fixed bad image quality on category view

> - Fixed acccessibility of the login button name

Reviewer: ```suggestion
- Fixed acccessibility of the login button
```
